Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,18 @@
 #include "BatteryStateOfChargeService.h"
-
+#include "BatteryData.h"
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    const double MS_HOUR_CONVERSION = 3600000.0;
+    const int PERCENT_TO_DECIMAL = 100;
 }
 
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
```

Comment:
Use an [initializor list](https://en.cppreference.com/w/cpp/language/initializer_list) for private variables in a constructor.